sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/ManagementChannel.java

```diff
@@ -38,6 +43,10 @@
 import static com.azure.messaging.servicebus.implementation.ManagementConstants.REQUEST_RESPONSE_MESSAGE_COUNT;
 import static com.azure.messaging.servicebus.implementation.ManagementConstants.SERVER_TIMEOUT;
 import static com.azure.messaging.servicebus.implementation.ManagementConstants.UPDATE_DISPOSITION_OPERATION;
+import static com.azure.messaging.servicebus.implementation.ManagementConstants.REQUEST_RESPONSE_LOCKTOKENS;
```

Comment:
Prefixing these constants with request_response isn't necessary. It was because all their message constants were in the same file. Ones relevant to Management are in the ManagementConstants file.

Reply:
LOCKTOKENS -> LOCK_TOKENS